### ISSUE TITLE
Compound types, e.g. Complex<f64> do not implement the Num trait

### ISSUE THREAD

**Issue by beneills** [rust-num/num#209]
I have been trying to create matricies with elements of type `Complex<f64>`.

I'm using the `la::Matrix` struct, but this library requires the element type to implement the `Num` trait.  This is not implemented for `Complex<f64>`.  Adding this impl to my code fails, and trying to incorporate it into a forked num crate seems non-trivial, since putting it in either _traits/_ or _complex/_ results in dependency issues.
# Reproductiion Steps

```
let m: Matrix<Complex<f64>> = Matrix::id(width, width);
error: the trait bound `num::Complex<f64>: num::Num` is not satisfied [E0277]
```

**Comment by cuviper** [comment]
You'll need to implement `Rem` and `Num`'s `fn from_str_radix`.

For the dependencies, it can only go in the `num-complex` crate.  What problem did you have with that?

**Comment by termoshtt** [comment]
I have encountered similar problem.
I am trying to implement `from_str_radix` for Complex64 using implementation for `f64`.

But I found a comment in impl macro of `Num`:
https://github.com/rust-num/num/blob/master/traits/src/lib.rs#L89

> FIXME: The standard library from_str_radix on floats was deprecated, so we're stuck
with this implementation ourselves until we want to make a breaking change.
(would have to drop it from `Num` though)

I hope that `from_str_radix` is splited from `Num` into another trait.
